src/components/ui/button/button.scss

```diff
@@ -35,7 +35,6 @@ $secondary-underline-offset-hover: 4px;
   box-shadow: none;
   color: var(--color-button__text);
   cursor: pointer;
-  display: flex;
```

Comment:
We need this, otherwise it alters the `Apply changes and close` button:

<img width="316" alt="Screenshot 2023-06-30 at 11 45 58" src="https://github.com/kedro-org/kedro-viz/assets/16869061/07a930fc-ecc4-421b-abb5-65b9538625bf">